packages/plugins/i18n/admin/src/components/CMHeaderActions.tsx

```diff
@@ -203,6 +233,44 @@ const LocalePickerAction = ({
 
       const permissionsToCheck = currentLocaleDoc ? canRead : canCreate;
 
+      if (isAiAvailable) {
```

Comment:
this only checks if AI is globally enabled/available, but you're not checking if i18n AI specifically is enabled. If it's not I don't think we should show this UI. We already have the other document action to push users to enable it